tests/Makefile

```
@@ -34,7 +34,7 @@ run-conformance-tests: ## Run conformance tests
 	kubectl run -i conformance \
 		--image=$(CONFORMANCE_PREFIX):$(CONFORMANCE_TAG) --image-pull-policy=Never \
 		--overrides='{ "spec": { "serviceAccountName": "conformance" }  }' \
-		--restart=Never -- sh -c "go test -v . -tags conformance,experimental -args --gateway-class=$(GATEWAY_CLASS) \
+		--restart=Never -- sh -c "go test -v . -run TestConformance/TLSRouteSimpleSameNamespace -tags conformance,experimental -args --gateway-class=$(GATEWAY_CLASS) \
```

This should skip the test we don't expect to pass:

```suggestion
		--restart=Never -- sh -c "go test -v . -skip TestConformance/TLSRouteInvalidReferenceGrant -tags conformance,experimental -args --gateway-class=$(GATEWAY_CLASS) \
```